spec/helpers/ems_cluster_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,41 @@
+describe EmsClusterHelper::TextualSummary do
+  describe 'Creates correctly textual groups' do
+    before do
+      instance_variable_set(:@record, FactoryGirl.create(:ems_cluster))
+      allow(@record).to receive(:ha_enabled).and_return("something")
+      allow(@record).to receive(:ha_admit_control).and_return("something")
+      allow(@record).to receive(:drs_enabled).and_return("something")
+      allow(@record).to receive(:drs_automation_level).and_return("something")
+      allow(@record).to receive(:drs_migration_threshold).and_return("something")
```

Comment:
Do you really need the `and_return` part here? Mocking is expensive and if you can work with the default `nil`, removing the `and_return` calls can speed up the tests a little.

Reply:
I need to have not-nil value to pass this [conditional statement](https://github.com/ManageIQ/manageiq-ui-classic/blob/ddf6869ff94de1288cd9090b3b3f41b591302921/app/helpers/ems_cluster_helper/textual_summary.rb#L32-L33):
```
if @record.ha_enabled.nil? && @record.ha_admit_control.nil? && @record.drs_enabled.nil? &&
@record.drs_automation_level.nil? && @record.drs_migration_threshold.nil?
```
Or are there any better options to do that?

Reply:
nope, it's good

Reply:
okay, thanks! :+1: